board/health.h

```diff
@@ -16,7 +16,7 @@ struct __attribute__((packed)) health_t {
   uint8_t car_harness_status_pkt;
   uint8_t usb_power_mode_pkt;
   uint8_t safety_mode_pkt;
-  int16_t safety_param_pkt;
+  uint16_t safety_param_pkt;
```

Comment:
think uint32_t is better here and that's what your cereal PR has

Reply:
I have a TODO ;)